tools/html-version/bounded-context-canvas-template.html

```diff
@@ -351,5 +351,73 @@ <h1>Bounded Context Canvas (v5)</h1>
                 </section>
             </article>
         </footer>
+
+        <script type="module">
+let downloadLink;
+
+document.body.addEventListener("change", ev => {
+    // synchronize form elements' properties with corresponding HTML state
+    let el = ev.target;
+    let tag = el.localName;
+    switch(tag) {
+    case "input":
+        switch(el.type) {
+        case "text":
+            el.setAttribute("value", el.value);
+            break;
+        case "radio":
+            let nodes = document.querySelectorAll(`input[type=radio][name=${el.name}`);
+            for(let node of nodes) {
+                if(node === el) {
+                    node.setAttribute("checked", "");
+                } else {
+                    node.removeAttribute("checked");
+                }
+            }
+            break;
+        default:
+            console.error(`WARNING: unsupported element \`${tag}\``);
+            break;
+        }
+        break;
+    case "textarea":
+        el.textContent = el.value; // XXX: is `#textContent` correct here?
+        break;
+    default:
+        console.error(`WARNING: unsupported element \`${tag}\``);
+        break;
+    }
+
+    // generate download link
+    if(downloadLink) {
+        downloadLink.remove();
+    }
+    let html = serializeDOM();
+    downloadLink = generateDownloadLink("bounded-context-canvas.html", html);
+    downloadLink.textContent = "download";
+    document.body.appendChild(downloadLink);
+});
+
+// adapted from TiddlyWiki <http://tiddlywiki.com>
```

Comment:
/cc @jermolene, just in case you're curious about the continuing influence of your creation...